packages/material-ui/src/utils/createSvgIcon.js

```diff
@@ -5,19 +5,19 @@ import SvgIcon from '../SvgIcon';
  * Private module reserved for @material-ui/x packages.
  */
 export default function createSvgIcon(path, displayName) {
-  const Component = React.memo(
-    React.forwardRef((props, ref) => (
-      <SvgIcon data-mui-test={`${displayName}Icon`} ref={ref} {...props}>
-        {path}
-      </SvgIcon>
-    )),
+  const Component = (props, ref) => (
+    <SvgIcon data-mui-test={`${displayName}Icon`} ref={ref} {...props}>
```

Comment:
Not really related to the diff in this PR, but what do we need `memo` for exactly? Was it added to prevent unnecessary re-renders or something?

Reply:
It's one of the only components that doesn't accept a react element as a prop, so one of the few spots where `memo` can work on the library. Do we have a benchmark that shows the win on a real-life use case? No.

Reply:
It's most likely worse for perf since SvgIcon almost never has an expensive tree on which we want to bail out. But now we have it so somebody might rely on it.